cli/install/template.go

```diff
@@ -940,37 +941,37 @@ spec:
             path: /ready
             port: 9995
           failureThreshold: 7
-        {{- if .EnableHA }}
+        {{- if .Values.EnableHA }}
         resources:
           requests:
             cpu: 20m
             memory: 50Mi
         {{- end }}
         securityContext:
-          runAsUser: {{.ControllerUID}}
+          runAsUser: {{.Values.ControllerUID}}
       volumes:
       - name: webhook-secrets
         secret:
-          secretName: {{.ProxyInjectorTLSSecret}}
+          secretName: {{.Values.ProxyInjectorTLSSecret}}
           optional: true
       - name: proxy-spec
         configMap:
           name: linkerd-proxy-injector-sidecar-config
 
----
 ### Proxy Injector Service Account ###
+---
```

Comment:
Having the `---` after the comment results in the comment not being included in the final output. You did the same in other places, but only here is the comment being stripped out.

Reply:
Yep this seemed to be an effect of the template engine consuming the next comment if the previous resource ends with a `status: {}`. The entire `template.go` is a mix of comments being consumed or displayed like this.

When I changed all the template comments to appear above the `---`, this was the result of me doing that. I'm not sure why it happens; I didn't investigate too much since we are modularizing these templates soon anyways (hopefully).

To reduce the diff, I did revert that change so thanks for pointing it out. It was not necessary!